conf/docker/dataverse-glassfish/init-container/reference_data.sql

```diff
@@ -0,0 +1,51 @@
+-- using http://dublincore.org/schemas/xmls/qdc/dcterms.xsd because at http://dublincore.org/schemas/xmls/ it's the schema location for http://purl.org/dc/terms/ which is referenced in http://swordapp.github.io/SWORDv2-Profile/SWORDProfile.html
```

Comment:
How is `conf/docker/dataverse-glassfish/init-container/reference_data.sql` different than `scripts/database/reference_data.sql` and why does it exist? We don't want to maintain two copies of this file.

Reply:
@thaorell any thoughts on this? We don't want to maintain two copies of this SQL file.